addons/io_scene_gltf2/blender/com/gltf2_blender_material_helpers.py

```diff
@@ -20,6 +20,8 @@ def get_gltf_node_name():
 def create_settings_group(name):
     gltf_node_group = bpy.data.node_groups.new(name, 'ShaderNodeTree')
     gltf_node_group.inputs.new("NodeSocketFloat", "Occlusion")
+    thicknessFactor  = gltf_node_group.inputs.new("NodeSocketFloat", "Thickness")
+    thicknessFactor.default_value = 1.0
```

Comment:
/cc @emackey I wonder if the default here should be 0.0? We will probably want a very explicit way of deciding when to export...

- (A) transmission + volume
- (B) only transmission

... and I think, (1) thickness is the most obvious condition, and (2) defaulting to 0 probably makes sense. We could also use the refraction depth property for the scalar, then we only need the settings node for a thickness _texture_.

Reply:
I used this as reference : https://github.com/KhronosGroup/glTF-Blender-IO/issues/1454#issuecomment-928319444

> But for glTF import/export, this may need to be another dead input on the glTF Settings node, like Occlusion. If the value is unknown, it's important to at least put 1.0 as the thicknessFactor, otherwise the volumetric effects are disabled.

But I maybe misunderstand it

Reply:
I see! Ok, yeah if we're enabling KHR_materials_volume when a "Volume Absorption" node is attached, as I think @emackey is suggesting, that seems like a good choice. Feel free to ignore my suggestion above. 👍🏻

Reply:
Yes. The glTF PBR model now has two distinct modes, "thin walled" for polygon shells, and "thick volumetric" for polygon meshes that enclose a manifold volume of material.  When `thicknessFactor` is zero, we're in thin-walled mode, and when it's nonzero we're in volumetric mode.  The actual nonzero value will be ignored by path tracers, but can be used by real-time rasterizers to estimate the thickness of material that the light will traverse.